### ISSUE TITLE
experiment names

### ISSUE THREAD

**Issue by taylor13** [WCRP-CMIP/CMIP7-CVs#2]
Found a difference between the experiment name listed in https://github.com/WCRP-CMIP/CMIP7_CVs/blob/main/src/activityId-experimentId.txt and the CMIP6 names.  Is this a problems?

Should esm-historical be esm-hist?

**Comment by durack1** [comment]
@taylor13 good catch, once CVs/jsons are populated in this repo this correction can be applied

**Comment by martinjuckes** [comment]
We also have these terms in the data request. Can we get a corrected list for the fast track in place next week? The draft CMIP7 fast track paper appears to be sticking with `esm-hist`. 

Are there other differences?

**Comment by durack1** [comment]
Just corrected the `esm-hist` typo, and augmented list with the full suite of current AerChemMIP exps.

Agreed @martinjuckes it will be good to convert these known experiments, and their host activity_id's/MIPs to json CVs so we can start to process of centralizing info
